release-notes-1.1.0.md

```diff
@@ -4,6 +4,7 @@ ${version-number}
 
 #### New Features
 - **SCMOD-9780**: Updated images to use Java 11
+- **SCMOD-9861**: Updated to ignore unsupported mapping changes
```

Comment:
I think I'd count this as a breaking change.  Let's move to 2.0.0-SNAPSHOT for this.